src/solidlsp/language_servers/intelephense.py

```diff
@@ -123,12 +126,21 @@ def _get_initialize_params(repository_absolute_path: str) -> InitializeParams:
                 }
             ],
         }
-
+        initialization_options = {}
         # Add license key if provided via environment variable
         license_key = os.environ.get("INTELEPHENSE_LICENSE_KEY")
         if license_key:
-            initialize_params["initializationOptions"] = {"licenceKey": license_key}
+            initialization_options["licenceKey"] = license_key
+
+        custom_intelephense_settings = self._solidlsp_settings.ls_specifics.get(self.__class__.__name__, {})
```

Comment:
We originally discussed that we would use the enum item's value that is associated with the language server.
Do you think the class name is a better option?
Right now, the `Language` enum is actually a `LanguageServer` enum - and perhaps we should rename it accordingly - and use it accordingly.

Reply:
The enum value would be preferable, because the user is already familiar with the respective values.
And we should probably document the possible options in `serena_config.yml` to make this accessible.